Request to merge branch rohit to develop

ukirderohit opened the pull request:
Task 1 Complete: 
1. Fix Button component mistake and fix that
2. Not imported and CSS
3. Create a component called label
4. Film Card
Separate commits were made for every addition

ukirderohit committed:
Fixed Button Component Mistake

Default props sizes were not working when changed

ukirderohit committed:
Integrating Antd CSS

ukirderohit committed:
Added Label and Title Component as part of typography

ukirderohit committed:
Added Image and Card component to support Film card molecule

ukirderohit committed:
Added FilmCard Component

ukirderohit committed:
Integrated Label, Image, Filmcard Component to test in im-ott App

pranay2401 commented on packages/im-components/src/atoms/button/constants/button.sizes.js:
```diff
@@ -1,6 +1,6 @@
-const MEDIUM = "MEDIUM";
-const LARGE = "LARGE";
+const SMALL = "small";
+const MEDIUM = "middle";
```
Is it "middle" or "default"? I tried with "default" for Medium string.

ukirderohit replied:
If you check size here you can see it has three options available (small,middle,large) 
default is middle
https://ant.design/components/button/#API

ukirderohit committed:
Spacing and Code Review Changes

ukirderohit committed:
Added application skeleton with navbar and footer from antd

ukirderohit committed:
React Router Integrated

ukirderohit committed:
Banner or Hero Section Added to landing page

ukirderohit committed:
Added lock file to gitignore

ukirderohit commented:
Task 2: 
1) Added Routing
2) Added Layout with Navbar, Component, Footer
3) Added Hero Section / Banner
Pending :
1) Right Scrolling Film Cards with Category
